### ISSUE TITLE
Repair readme example with evidence

### ISSUE THREAD

**Issue by slarson** [openworm/PyOpenWorm#92]
Readme had examples with the Evidence class.  Currently 90% of it doesn't work or isn't helpful to the user as an initial example.  

To repair, let's add some useful evidence into the database.  This would appear to relate to #75 but I think that is a larger issue from what I understand.  From the way the example described below works, we should be able to build an example around the provenance of the facts we are already putting into the database, like the fact that the connections between neurons come from the NeuronConnect.xls hosted at WormAtlas (as added by #89)

So specifically, our new example, rather than showing how the user puts evidence in (we can show that elsewhere), should only show how users get evidence OUT (higher priority for the average user), and what they should be able to look up is how they figure out what evidence exists that Neuron A is connected to Neuron B, or that Neuron C is connected to Muscle X.

IN FACT, I just realized that the documentation on retrieving evidence over here is perfect:

http://pyopenworm.readthedocs.org/en/alpha0.5/evidence.html#retrieving-evidence

It just doesn't work yet...  We have to implement #89 

PREVIOUS BLOCK FROM README:

Returns provenance information providing evidence about facts::

```python
  >>> ader = P.Neuron('ADER')
  >>> ader.receptors()
  set(['ACR-16', 'TYRA-3', 'DOP-2', 'EXP-1'])

  #look up what reference says this neuron has a receptor EXP-1
  >>> e = P.Evidence()
  >>> e.asserts(P.Neuron('ADER').receptor('EXP-1')) 
  asserts=receptor=EXP-1
  >>> list(e.doi())
  ['10.100.123/natneuro']

```

Add provenance information::

```python
  >>> e = P.Evidence(author='Sulston et al.', date='1983')
  >>> e.asserts(P.Neuron(name="AVDL").lineageName("AB alaaapalr"))
  asserts=lineageName=AB alaaapalr
  >>> e.save()

```

See what some evidence stated::
```python
  >>> e0 = P.Evidence(author='Sulston et al.', date='1983')
  >>> list(e0.asserts())
  [Neuron(name=AVDL,lineageName=AB alaaapalr)]

```

**Comment by mwatts15** [comment]
Made a small demo for the examples: https://gist.github.com/mwatts15/3600df9be5d2d00e366e
The command line toggle is to skip my local database and read in the OpenWormData.n3 file.

**Comment by slarson** [comment]
Fixed by #125
